### DISCUSSION TITLE
Fill space between 2 line charts with colour

### DISCUSSION THREAD

**Discussion by aikastor** [airbnb/visx#1538] [Q&A]
So I have set up two line charts, and what I need is the space between them to be filled with color. I've tried using the Threshold component, but I do not think that this is the right component for that, any idea how to achieve this?

here's a link to sandbox: https://codesandbox.io/s/unruffled-resonance-5hbs58?file=/src/App.js
here is the expected outcome 
<img width="510" alt="Screen Shot 2022-07-11 at 01 44 43" src="https://user-images.githubusercontent.com/33369564/178160545-a1e1febd-8c76-4b87-bc69-5a62c4991b94.png">

**Comment by williaster** [comment]
Hi @aikastor 👋  . Based on an initial understanding I'd say that the Threshold component is the best way to achieve this, your codesandbox looks good to me. Is that implementation lacking in some way/can you elaborate on why you don't think it's the right component?
